shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -302,8 +369,8 @@ Meteor.publish("powerboxOptions", function (requestId, descriptorList) {
   // packed format and base64-encoded. The publish populates a pseudo-collection called
   // `powerboxOptions`. Each item has the following fields:
   //
-  //   _id: Object with two fields: `requestId` (as passed when subscribing) and `optionId` (a
-  //       unique identifier for the option).
+  //   _id: Unique identifier string.
```

Comment:
Oh, I see, you are changing the comment to match the implementation. But in fact the implementation is wrong and the comment is right... >_>